docs/version_releases.rst

```diff
@@ -0,0 +1,20 @@
+Version releases
+================
+
+Create new release
+------------------
+
+How create new release you can read `this <https://help.github.com/en/articles/creating-releases>`_.
+
+Requirements
+------------
+- New release should based on master branch.
+- New release tag must have name something like 1.0.0.
+
+A normal version number MUST take the form X.Y.Z where X, Y, and Z are non-negative integers, and MUST NOT contain leading zeroes.
```

Comment:
It doesn't explain what major/minor and patch versions mean, I would rather just copy/paste a text from summary section on semver.org (https://semver.org/#summary).
Also I would include a link to semver site for more information.

Reply:
Changed.